objects/src/testing/account.rs

```diff
@@ -59,8 +59,8 @@ impl<T: Rng> AccountBuilder<T> {
         self
     }
 
-    pub fn add_storage_slot(mut self, slot: StorageSlot) -> Self {
-        self.storage_builder.add_slot(slot);
+    pub fn add_storage_slot(mut self, slot: StorageSlot, location: Option<usize>) -> Self {
```

Comment:
Similar question to one of the previous comments: why do we need to add `location` here? Is there a way to avoid it?

Reply:
Answered here: https://github.com/0xPolygonMiden/miden-base/pull/875#discussion_r1761794005